src/tribler/core/components/metadata_store/db/orm_bindings/torrent_state.py

```diff
@@ -29,4 +31,8 @@ def to_health(self) -> HealthInfo:
             return HealthInfo(infohash=self.infohash, last_check=self.last_check,
                               seeders=self.seeders, leechers=self.leechers)
 
+        @property
+        def seeders_leechers_last_check(self) -> Tuple[int, int, int]:
```

Comment:
NIT: maybe it is better to replace Tuple with Dataclass? It would be more obvious in terms of guessing what is [int, int, int].